packages/accounts/src/light-account/types.ts

```diff
@@ -68,7 +72,7 @@ export type GetLightAccountVersionDefsForType<TType extends LightAccountType> =
 
 export type GetLightAccountVersionDef<
```

Comment:
same q here

Reply:
Because LightAccountVersionDef is already defined as actual type definition for type with input type param being all specified. This type helper infers or extracts the lightaccountversiondef that is tightly coupled with the input type param .